source/simulator/parameters.cc

```diff
@@ -1668,6 +1698,27 @@ namespace aspect
         }
     }
     prm.leave_subsection ();
+
+    prm.enter_subsection ("Boundary heat flux model");
+    {
+      try
+        {
+          const std::vector<types::boundary_id> x_fixed_heat_flux_boundary_indicators
+            = geometry_model.translate_symbolic_boundary_names_to_ids(Utilities::split_string_list
+                                                                      (prm.get ("Fixed heat flux boundary indicators")));
+          fixed_heat_flux_boundary_indicators
+            = std::set<types::boundary_id> (x_fixed_heat_flux_boundary_indicators.begin(),
+                                            x_fixed_heat_flux_boundary_indicators.end());
+        }
+      catch (const std::string &error)
+        {
+          AssertThrow (false, ExcMessage ("While parsing the entry <Boundary heat flux model/Fixed heat flux "
+                                          "boundary indicators>, there was an error. Specifically, "
```

Comment:
I don't like "there was an error": can you try to rephrase?

Reply:
That's how it's done for all the other boundary indicators (Free surface, traction, etc.). Do you want me to change all of them? Maybe we just leave it as is?

Reply:
then leave as is.